pkg/reconciler/route/traffic/rollout.go

```diff
@@ -451,13 +451,13 @@ func (cur *ConfigurationRollout) computeProperties(nowTS, minStepSec, durationSe
 	// at 1s, so we just do 1 step.
 	durationSecs = math.Max(1, durationSecs-minStepSec)
 	numSteps := math.Max(1, durationSecs/minStepSec)
-	pf := float64(cur.Percent)
 
 	// The smallest step is 1%, so if we can fit more steps
-	// than we have percents, we'll make c.Percent-1 steps
+	// than we have percents, we'll make pf = c.Percent-1 steps
 	// each equal to 1%. -1, since we already moved 1% of the traffic.
+	pf := float64(cur.Percent - 1)
```

Comment:
maybe `pf` -> `remainingPercent`, since (a) it's not "percent as float" any more :) and (b) I personally found it a bit tricky to think this logic through and having the definition of pf more clear would help I think

Reply:
sure renamed.